src/pages/FilterMenu.css

```diff
@@ -117,18 +117,23 @@ ul.FilterMenu-trendy li.FilterMenu-trendy-title {
   color: rgba(255, 255, 255, 0.5);
 }
 
-.FilterMenu-form input.FilterMenu-valid {
-  display: none;
-  border: 0px;
-  width: 70px;
-  height: 70px;
-  font-size: 3em;
-  line-height: 1.5em;
-  text-align: center;
-  color: #FFFFFF;
-  border-radius: 35px;
-  background: url(../img/btn-check.png) no-repeat center center;
-  order: 0;
-  flex: 0 1 auto;
-  align-self: center;
+.FilterMenu-valid {
```

Comment:
that is a button so maybe just call it checkButton or so but maybe not valid. I would assume valid to be a class that handles the style of a input field once it is valid or something.